src/Lib.hs

```diff
@@ -166,7 +166,7 @@ runPoolInsertion poolMetadataJsonPath poolId poolHash = do
 
     let addPoolMetadata = dlAddPoolMetadata dataLayer
 
-    addPoolMetadata poolId poolHash poolMetadataJson (pomTicker decodedMetadata)
+    addPoolMetadata (panic "runPoolInsertion") poolId poolHash poolMetadataJson (pomTicker decodedMetadata)
```

Comment:
This doesn't look right.

Reply:
Its a stub for the non-PorstgreSQL later. That field does not make sense for the non-Postgres layer.

Reply:
This is not a stub. 
It is called if you call `SMASHPGPASSFILE=config/pgpass stack run smash-exe -- insert-pool --metadata test_pool.json --poolId "062693863e0bcf9f619238f020741381d4d3748aae6faf1c012e80e7" --poolhash "cbdfc4f21feb0a414b2b9471fa56b0ebd312825e63db776d68cc3fa0ca1f5a2f"`.

Reply:
Fixed.